pkg/templateservicebroker/servicebroker/servicebroker.go

```diff
@@ -47,6 +55,13 @@ func NewBroker(saKubeClientConfig *restclient.Config, informer templateinformer.
 	configCopy := *saKubeClientConfig
 	configCopy.NegotiatedSerializer = serializer.DirectCodecFactory{CodecFactory: kapi.Codecs}
 
+	delay := 5
+	value := os.Getenv("TEMPLATE_SERVICE_BROKER_GC_DELAY_SECONDS")
```

Comment:
how about `TEMPLATE_SERVICE_BROKER_GC_DELAY` which is a duration string parseable with `time.ParseDuration`? More flexible, plus it looks like right now it looks like you're passing the `int` through directly as a `time.Duration` which I believe will treat your unit as nanosecond.